src/components/turnstile/_turnstile.scss

```diff
@@ -0,0 +1,60 @@
+/* ===============Turnstile================== */
+.cmp-adaptiveform-turnstile{
+    color: $dark-gray;
+    font-size: $font-md;
+    margin: $field-margin;
+    display: flex;
+    flex-direction: column;
+  }
+  .cmp-adaptiveform-turnstile__widget{
+  
+  }
+  .cmp-adaptiveform-turnstile__longdescription {
+    color: $light-gray;
+    background-color: $very-light-gray;
+    font-size: $font-sm;
+    margin-top: $error-top-margin;
+    margin-bottom: 5px;
+    padding: 10px;
+    p{
+      margin: 0;
+      padding: 0;
+    }
+  }
+  .cmp-adaptiveform-turnstile__shortdescription {
+    font-size: $font-sm;
+    margin-top: $error-top-margin;
+  }
+  .cmp-adaptiveform-turnstile__questionmark {
+    display:inline-block;
+    width:1rem;
+    height:1rem;
+    border-radius: 9px;
+    background: url(./resources/images/question.svg) center center / cover no-repeat,#969696;
+    cursor : pointer;
+    // below properties for backward compatibility
+    position: absolute;
+    right: 20px;
+  }
+  // below properties for backward compatibility
+  .cmp-adaptiveform-turnstile__label-container .cmp-adaptiveform-turnstile__questionmark{
+    position: unset;
+    right: unset;
+  }
```

Comment:
Since it doesn't have a label, please check once if the question mark is rendering fine on the right in normal mode.
And on left in rtl mode. In other components, there is label-container css which takes care of rtl behavior between label and question mark